### ISSUE TITLE
Should `async_resources` be updated manually when other async components are added?

### ISSUE THREAD

**Issue by jlfsjunior** [plotly/dash-component-boilerplate#136]
Hi,

I started trying my way around async components and I was having some issues when I added a second component to the library. To reproduce it, just copy the one created by cookiecutter, refactor the name to something else and export it in the right places...

The issue was somehow related to the browser not finding async-NewComponent*.js, so I realized that `__init__.py` has an `async_resources` list containing just the component created automatically by cookiecutter.

As I asked in the title, should this list be modified manually or is it expected that `dash-generate-components` should do the trick?

Thanks!

**Comment by alexcjohnson** [comment]
Good point @jlfsjunior - yes, it's required to manually add new async components to `__init__.py`, there is nothing that automatically updates this file right now.
